site/pages/_document.page.tsx

```diff
@@ -16,6 +16,11 @@ const generateCsp = (): { csp: string; nonce: string } => {
     const csp = `
         default-src 'self';
         script-src 'self' 'nonce-${nonce}' 'strict-dynamic' ${production ? "" : "'unsafe-eval' 'unsafe-inline'"};
+        connect-src 'self'  ${
+            !production
+                ? "https://api.test.ref-data.dft-create-data.com/"
```

Comment:
Same thing here around using an env var for the api url

Reply:
done